package.json

```diff
@@ -168,7 +168,8 @@
     "tinyliquid": "^0.2.33",
     "url-loader": "^0.5.7",
     "uswds": "0.10.0",
-    "webpack": "^1.13.1",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#3d7c8e8b47443dfd2b494a3a4151aa2afaa63118",
```

Comment:
This seems out of date? The commit should be ee444b33dd221524c71b3b0201beda250a0ffd65, based on what I committed this morning.